packages/fxa-auth-server/lib/senders/emails/templates/cadReminderSecond/index.stories.ts

```diff
@@ -12,7 +12,7 @@ export default {
 
 const createStory = storyWithProps(
   'cadReminderSecond',
-  'Sent 72 hours after a user clicks "send me a reminder" on the connect another device page.',
+  'Sent 72 hours after a user clicks "send me a reminder" on the connect another device page. At the time of writing, it cannot be triggered in staging, only in production (FXA-4515).',
```

Comment:
Small nit, "only in production" kind of makes it sound like it can't be triggered locally either, but maybe that's the case?

Do you mind commenting on that issue to say when it's fixed, this sentence should be removed?